### PR TITLE
Milestone 1 "Work" section

### PR DESCRIPTION
**Pull request by Gmanlove**
### **In this milestone, I implemented the following requirements**

- [ ]  I used html and css to create my portfolio website
- [ ]  I installed linters configurations and made sure there are no linters error
- [ ] Build the work sections of the mobile website using the template chosen in the previous step.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,68 +1,286 @@
...
+                  </li>
+                </ul>
+              </div>
+            </li>
+            <li>
+              <div class="know">
+                <p class="know-text flex">
+                  <span class="text">Frameworks</span
+                  ><span class="arrow">></span>
+                </p>
```
Comment (bienvenuushindi) [inline index.html:265]: [OPTIONAL] I think the parent tag here should be div and not p tag. The <p> tag defines a paragraph  while the DIV tag is simply an HTML container. Its only purpose is to let you group together other document elements to which to apply the same CSS style

File: index.html [hunk 2]
```diff
@@ -1,68 +1,286 @@
...
+              </div>
+            </li>
+            <div class="arrow-line">
+              <hr />
+            </div>
+            <li>
+              <div class="know">
+                <p class="know-text flex">
+                  <span class="text">Skills</span><span class="arrow">></span>
+                </p>
```
Comment (bienvenuushindi) [inline index.html:275]: [OPTIONAL] I think the parent tag here should be div and not p tag. The <p> tag defines a paragraph  while the DIV tag is simply an HTML container. Its only purpose is to let you group together other document elements to which to apply the same CSS style

File: index.html [hunk 3]
```diff
@@ -1,68 +1,286 @@
...
+          <button type="button">Get my resume</button>
+        </div>
+        <div class="knowdge flex column">
+          <ul>
+            <li>
+              <div class="know flex column">
+                <p class="know-text flex">
+                  <span class="text">Languages</span>
+                  <span class="know-line">></span>
+                </p>
```
Comment (bienvenuushindi) [inline index.html:243]: [OPTIONAL] I think the parent tag here should be div and not p tag. The <p> tag defines a paragraph  while the DIV tag is simply an HTML container. Its only purpose is to let you group together other document elements to which to apply the same CSS style

File: index.html [hunk 4]
```diff
@@ -1,68 +1,286 @@
...
+                <li>
+                  <a href="#"> <img src="images/github.png" alt="" /> </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img src="images/Linkedin icon.svg" alt="" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/angellist.png" alt="" /> </a>
```
Comment (bienvenuushindi) [inline index.html:223]: [OPTIONAL] Kindly make sure you add some text to  ALT attribute. 
ALT text refers to invisible description of images which are read aloud to blind users on a screen reader. Adding ALT text allows authors to include images, but still provide the content in an alternative text based format

File: index.html [hunk 5]
```diff
@@ -1,68 +1,286 @@
...
+                </li>
+                <li>
+                  <a href="#"> <img src="images/twitter.png" alt="" /> </a>
```
Comment (bienvenuushindi) [inline index.html:226]: [OPTIONAL] Kindly make sure you add some text to  ALT attribute. 
ALT text refers to invisible description of images which are read aloud to blind users on a screen reader. Adding ALT text allows authors to include images, but still provide the content in an alternative text based format

**Review by bienvenuushindi — APPROVED** [review]
# Approved 👏  

## Hi @Gmanlove   👋, 

Well done implementing the project requirements 👏

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @bienvenuushindi  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -1,68 +1,286 @@
...
+                </li>
+                <li>
+                  <a href="#"> <img src="images/medium.png" alt="" /> </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/github.png" alt="" /> </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/angellist.png" alt="" /> </a>
+                </li>
```
Comment (Gmanlove) [inline index.html:58]: - [ ] i kindly sugest t

**Review by Gmanlove — COMMENTED** [review]
.

**Pull request merged**